src/kixi/lapita/schemas.clj

```diff
@@ -12,12 +12,11 @@
    Returns a map which separates coerced and non-coerced data."
   [data schema]
   (let [data-coercer (coerce/coercer schema coerce/string-coercion-matcher)]
-    (->> data
-         (map (fn [d] (let [coerced (data-coercer d)]
-                        (if (su/error? coerced)
-                          {:non-coerced (list d)}
-                          {:coerced (list coerced)}))))
-         (apply merge-with concat))))
+    (map (fn [d] (let [coerced (data-coercer d)]
+                   (if (su/error? coerced)
+                     (assoc d :status :non-coerced)
```

Comment:
Is this going to play havoc if there is already a column in the CSV named "status" ?